cmd/podman/varlink/io.podman.varlink

```diff
@@ -1035,6 +1035,22 @@ method GenerateKubeService() -> (notimplemented: NotImplemented)
 # like that created by GenerateKube. See also [GenerateKube](GenerateKube).
 method ReplayKube() -> (notimplemented: NotImplemented)
 
+# ContainerConfig returns a container's config in string form. This call is for
```

Comment:
I'm honestly OK with people using this one remotely - it seems sort of useful, even if it's a JSON string and not a literal varlink struct